dll/win32/credui/lang/credui_Ro.rc

```diff
@@ -39,13 +40,13 @@ BEGIN
     CONTROL         IDB_BANNER,-1,"Static",SS_BITMAP | SS_CENTERIMAGE,0,
                     0,213,37
     LTEXT           "",IDC_MESSAGE,8,48,199,8,NOT WS_GROUP
-    LTEXT           "N&ume utilizator:",-1,8,62,72,12,SS_CENTERIMAGE
+    LTEXT           "Nume utilizator:",-1,8,62,72,12,SS_CENTERIMAGE
     CONTROL         "",IDC_USERNAME,"ComboBoxEx32",CBS_DROPDOWN |
                     CBS_NOINTEGRALHEIGHT | WS_TABSTOP,80,62,126,87
-    LTEXT           "&Parolă:",-1,8,80,72,12,SS_CENTERIMAGE
+    LTEXT           "&Parolă",-1,8,80,72,12,SS_CENTERIMAGE
```

Comment:
There is really no `:` here in the end on Windows? Can you share a pic of that dlg of the romanian MS-credui?

Reply:
Yes, there is. I didn't pay attention. Here is the screenshot.

![2024-04-13 11_31_09-](https://github.com/reactos/reactos/assets/113553683/c1c5af27-326e-4c8e-b9f9-304ece2b2730)

Reply:
I don't understand why immediately after I fixed the error I've got build errors.

Reply:
Resolved, the failing MSVC builders are unrelated to your PR. Maybe out of disk space. The do fail when linking vfd, which is a different module.